virt/virt-2-6-release-notes.adoc

```diff
@@ -66,7 +66,7 @@ The SVVP Certification applies to:
 [id="virt-2-6-storage-new"]
 === Storage
 
-* When you add a virtual machine disk to a persistent volume claim (PVC) that uses the `Filesystem` volume mode, the Containerized Data Importer (CDI) now reserves 5.5% of the PVC space for file system overhead. If the default value is not ideal for your use case, you can xref:../virt/virtual_machines/virtual_disks/virt-reserving-pvc-space-fs-overhead.adoc#virt-reserving-pvc-space-fs-overhead[change the percentage] that CDI reserves for this purpose.
+* When you add a virtual machine disk to a persistent volume claim (PVC) that uses the `Filesystem` volume mode, the Containerized Data Importer (CDI) now reserves 5.5% of the PVC space for file system overhead. 
```

Comment:
We probably shouldn't edit 2.6 release notes

Reply:
Eek that was unintentional. Thanks for the catch @maya-r